areas-of-d-usage.dd

```diff
@@ -322,4 +322,7 @@ Macros:
                 margin-top: -200px;
             }
         }
+        h2 {
```

Comment:
Any reason why this CSS is stored in this file? If not, it's probably a good idea to move this to style.css